releases/release-1.26/release-notes/release-notes-draft.md

```diff
@@ -8,33 +8,103 @@
 
 ### Deprecation
 
+- CLI flag `pod-eviction-timeout` is deprecated and will be removed together with `enable-taint-manager` in v1.27. ([#113710](https://github.com/kubernetes/kubernetes/pull/113710), [@kerthcet](https://github.com/kerthcet))
 - Kube-apiserver: the unused `--master-service-namespace` flag was deprecated and will be removed in v1.27. ([#112797](https://github.com/kubernetes/kubernetes/pull/112797), [@SataQiu](https://github.com/SataQiu))
 - The `gcp` and `azure` auth plugins have been removed from `client-go` and `kubectl`. See [kubelogin](https://github.com/Azure/kubelogin) and [Kubectl Auth Changes in GKE](https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke) for details about the cloud-specific replacements. ([#112341](https://github.com/kubernetes/kubernetes/pull/112341), [@enj](https://github.com/enj))
 
 ### API Change
 
-- <NONE> ([#86139](https://github.com/kubernetes/kubernetes/pull/86139), [@jasimmons](https://github.com/jasimmons))
+- 'A new `preEnqueue` extension point was added to scheduler's component config
```

Comment:
Why there are single quotes here for each item?

Reply:
Good catch! They shouldn't have quotes. Sometimes we add those single quotes if the first word starts with back-ticks (``) to avoid [a known issue](https://github.com/kubernetes/release/issues/2753). But these lines are not the case. I'll submit a PR to fix this.
Thanks for the review!